### PR TITLE
Enhancement: Examples/with-ledger

### PR DESCRIPTION
**Pull request by mahmud-bn**
### Description

Adds an example folder with a Next.js project that implements a basic @web3-onboard/ledger support

### Checklist
- [ ] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [ ] The box that allows repo maintainers to update this PR is checked
- [ ] I tested locally to make sure this feature/fix works
- [ ] I have run `yarn type-check` & `yarn build` to confirm there are not any associated errors
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

File: examples/with-ledger/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  if(wallet){
+    console.log(wallet)
+  }
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>Web3-Onboard + Ledger Demo</title>
+        <meta
+          name="description"
```
Comment (Adamj1232) [inline examples/with-ledger/pages/index.js:91]:
```suggestion
          name="Web3-Onboard Example with Ledger Hardware Wallet"
```

File: examples/with-ledger/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+          content="Example of how to integrate Web3-Onboard with Next.js"
```
Comment (Adamj1232) [inline examples/with-ledger/pages/index.js:92]:
```suggestion
          content="Example of how to integrate Web3-Onboard's Ledge Module with Next.js"
```

File: examples/with-ledger/pages/index.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    }
+  }
+})
+
+
+
+export default function Home() {
+  const [{ wallet, connecting }, connect, disconnect] = useConnectWallet()
+
+  if(wallet){
```
Comment (Adamj1232) [inline examples/with-ledger/pages/index.js:83]: Do we still want this log? Prettier-ify?

**Review by Adamj1232 — APPROVED** [review]
Overall looks good, might have some fun adding some styles to the landing page if you want 🤓

**Comment by mahmud-bn** [comment]
> Overall looks good, might have some fun adding some styles/BN logos to the landing page if you want 🤓

I'm not sure we want that across all examples including prettier. I would assume we already have a fancy React Demo for that. What we want, I think, is a set of examples that can be easily forked. 
E.g
https://github.com/wagmi-dev/wagmi/tree/main/examples/next
https://github.com/Uniswap/web3-react/tree/main/packages/example-next

**Review by aaronbarnardsound — APPROVED** [review]

**Pull request merged**